src/components/button/button_toggle/button_toggle.tsx

```diff
@@ -41,16 +41,18 @@ export interface EuiButtonToggleProps extends EuiButtonProps {
    * Used primarily for `EuiButtonGroup`
    */
   type?: ToggleType;
+
+  onChange?: React.FormEventHandler<HTMLInputElement>;
```

Comment:
`onChange` below is being applied to the `input` element but the `onChange` prop was inheriting from both `HTMLAnchorElement` and `HTMLButtonElement`.